Model/Content.php

```diff
@@ -167,6 +167,27 @@ public function __construct()
     }
 
     /**
+     * For purpose of entity cloning
+     */
+    public function __clone()
```

Comment:
Doctrine's generated proxy class implements a standard __clone() which unsets the identifier and the persister. There's no need to implement this if you only want to unset the identifier. There's also no need to detach the entity after cloning if you don't override this method.